src/pages/404.tsx

```diff
@@ -0,0 +1,42 @@
+import { getMessages } from '../utils/locale';
+import { GetStaticProps } from 'next';
+import Head from 'next/head';
+import { FormattedMessage as Msg } from 'react-intl';
+import { Content, Flex, Heading, Link } from '@adobe/react-spectrum';
```

Comment:
This is a minor thing, but we try to separate library imports from imports that are from the project codebase. Library imports come first, and then separated by an empty line comes local imports.